packages/terra-paginator/src/Paginator.scss

```diff
@@ -0,0 +1,112 @@
+        text-decoration: var(--terra-paginator-nav-link-text-decoration-hover, underline);
+      }
+
+      &:focus {
+        background: var(--terra-paginator-nav-link-background-focus);
+        border: var(--terra-paginator-nav-link-border-focus);
+        box-shadow: var(--terra-paginator-nav-link-box-shadow-focus, 0 0 0 2px rgba(76, 178, 233, 0.25), inset 0 0 0 1px #4cb2e9);
+        outline: none;
+      }
+
+      &:active {
+        background: var(--terra-paginator-nav-link-background-active, rgba(0, 0, 0, 0.07));
+        color: var(--terra-paginator-nav-link-foreground-color-active, #001f67);
+        text-decoration: var(--terra-paginator-nav-link-text-decoration-active, underline);
+      }
+    }
+
+    .is-selected {
+      color: var(--terra-paginator-nav-link-selected-color, initial);
+      cursor: initial;
+      pointer-events: none;
+
+      &:hover {
+        text-decoration: none;
+      }
+
+      &:focus {
+        outline: none;
+      }
+    }
+
+    .is-disabled {
+      color: var(--terra-paginator-nav-link-foreground-color-disabled, rgba(28, 31, 33, 0.2));
+      cursor: default;
+      pointer-events: none;
+
+      &:focus {
+        box-shadow: var(--terra-paginator-nav-link-disabled-box-shadow-focus, initial);
+        outline: none;
+      }
+    }
+
+    .left-controls {
+      margin-left: var(--terra-paginator-nav-link-left-controls-margin-left, 0.14286rem);
+      margin-right: var(--terra-paginator-nav-link-left-controls-margin-right, 0.57143rem);
+    }
+
+    .right-controls {
+      margin-left: var(--terra-paginator-nav-link-right-controls-margin-left, 0.57143rem);
+      margin-right: var(--terra-paginator-nav-link-right-controls-margin-right, 0.14286rem);
+    }
+  }
+
+  .icon {
+    display: inline-block; // IE 10
+    height: 1rem;
+    vertical-align: middle;
+    width: 1rem;
+  }
+
+  .previous > .icon {
+    content: var(--terra-paginator-nav-link-icon-previous, inline-svg('<svg viewBox="0 0 48 48" xmlns="http://www.w3.org/2000/svg"><path fill="#0059a8" d="M10.3 24L33.8 0l3.9 3.8L18 24l19.7 20.2-3.9 3.8z"></path></svg>'));
+    margin-right: 0.25rem;
+  }
+
+  .previous.is-disabled > .icon {
+    content: var(--terra-paginator-nav-link-icon-previous-disabled, inline-svg('<svg viewBox="0 0 48 48" xmlns="http://www.w3.org/2000/svg"><path fill="rgba(28, 31, 33, 0.2)" d="M10.3 24L33.8 0l3.9 3.8L18 24l19.7 20.2-3.9 3.8z"></path></svg>'));
+  }
+
+  .next > .icon {
+    content: var(--terra-paginator-nav-link-icon-next, inline-svg('<svg viewBox="0 0 48 48" xmlns="http://www.w3.org/2000/svg"><path fill="#0059a8" d="M37.7 24L14.2 48l-3.9-3.8L30 24 10.3 3.8 14.2 0z"></path></svg>'));
+    margin-left: 0.25rem;
+  }
+
+  .next.is-disabled > .icon {
+    content: var(--terra-paginator-nav-link-icon-next-disabled, inline-svg('<svg viewBox="0 0 48 48" xmlns="http://www.w3.org/2000/svg"><path fill="rgba(28, 31, 33, 0.2)" d="M37.7 24L14.2 48l-3.9-3.8L30 24 10.3 3.8 14.2 0z"></path></svg>'));
+  }
+}
```

Comment:
Need to add at the end:
```
.previous > .icon-only,
.next > .icon-only {
    margin-left: 0;
    margin-right: 0;
}
```

Reply:
https://github.com/cerner/terra-core/pull/1445/commits/37187bdacdbd135aef8fab5bf1ca8b24cbbc2bf9